_posts/en/2018-07-30-xapo-consolidation.md

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Field Report: Xapo UTXO Consolidation'
```

Comment:
This is a good, neutral headline, but if you want to maybe make it a bit catchier, I'd suggest: "Field Report: Consolidation of 4 Million UTXOs at Xapo", or anything else that mentions the large extent of the effort.